t/plugin/jwt-auth.t

```diff
@@ -1209,7 +1241,39 @@ hello world
 
 
 
-=== TEST 43: test for unsupported algorithm
+=== TEST 44: sign / verify (algorithm = HS512,with extra payload)
```

Comment:
Maybe you are misled by my example. We need two tests, one for default and another for RS256 (but not for HS512).

Reply:
Test 19 is for RS256.